src/iperf_time.h

```diff
@@ -34,6 +34,28 @@ struct iperf_time {
     uint32_t usecs;
 };
 
+/**
+ * Returns the current wallclock time.
```

Comment:
```suggestion
/**
 * Retrieve the current wallclock time.
```

It returns the success code of the operation. The time is in the out-parameter. Same for the monotonic function.

Reply:
good idea, thanks!